mix.exs

```diff
@@ -35,6 +35,7 @@ defmodule K8s.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
+      {:castore, ">= 0.0.0"},
```

Comment:
Why not ` {:castore, "~> 0.1.0"}`?

Reply:
I think you're right it'd probably be better to pin to the minor version. At the moment they are functionally equivalent all releases are `0.1.X`, but we could protect against any major API changes by pinning it to the minor version. Let me update that real quick.

Reply:
Updated 👍🏻